raddb/policy.d/eap

```diff
@@ -74,9 +71,7 @@ permit_only_eap {
 #
 remove_reply_message_if_eap {
 	if (&reply.EAP-Message && &reply.Reply-Message) {
-		update reply {
-			&Reply-Message !* ANY
-		}
+		&reply.Reply-Message !* ANY
```

Comment:
Same here.  `!*` is no longer allowed.  If it's not an error in the current parser, then it should be an error.